### ISSUE TITLE
Further Explanation

### ISSUE THREAD

**Issue by larkben** [alephium/alephium-nft#16]
contracts/nft.ral

what do you mean by it only supports 1 token. 1 token as in it couldn't be used for mass mint of the same nft or is it okay to use this for say an nft collection all different and minted separately?

**Comment by polarker** [comment]
UTXO model could support factional NFT which is semi-fungible. The standard NFT tokens have just 1 token.

**Comment by larkben** [comment]
Yes, that makes sense and running these contracts I'm noticing the checkCaller!() function is hinted at.. is there a particular reason it isn't used in these contracts or can I void it by setting it to false in the function params.
